### DISCUSSION TITLE
Origin of 4G modules

### DISCUSSION THREAD

**Discussion by kisPlus** [danielewood/sierra-wireless-modems#133] [Q&A]
There are a lot of 4G modules on the market with different type numbers.  I'm wondering do they originate all from EM/MC7455 type or other type of Sierra Wireless customized by other producers or there are other base makes of different producers (e.g. different types of Foxconn used by HP origin from Sierra Wireless, customized by Foxconn for HP or their (Foxcon) own makes). Can they be configured like Sierra Wireless, what about passwords, AT commands etc.

**Comment by danielewood** [comment]
Most will be standard in every way except three things:

1. VID/PID will likely be custom to the OEM.
2. They may have some magic that they do that lets them be recognized as an OEM whitelisted device, what that magic exactly is, I have never investigated.
3. They may use a custom password, but this is almost exclusively done for Sierra Wireless based hotspots, and not for the modules.

Any other type of customization requires constant engineering work by the OEM, so they tend to not do that.
